src/components/Containers/TopSection/TopSection.jsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import styled from "styled-components";
+
+import TitleGraphic from "../../../assets/home/top-section-graphic.svg";
+import { media } from "../../../utils";
+import { Text } from "../../Texts";
+import { Button } from "../../Inputs";
+
+export const TopSection = ({ ...props }) => (
+  <MainContainer {...props}>
+    <FirstColumn>
+      <JosefinTitle size="4.5em" bold="true">
+        Prospective Medical Professionals
+      </JosefinTitle>
+      <SText size="default">
+        A registered charity dedicated to connecting aspiring students to health
+        sciences opportunities
+      </SText>
+      <Button>
+        <ButtonText size="default" bold="true">
+          DONATE
+        </ButtonText>
+      </Button>
+    </FirstColumn>
+    <SecondColumn>
+      <Graphic
+        src={TitleGraphic}
+        alt="Title graphic - person with book, stethoscope, pills and syringe"
+      />
+    </SecondColumn>
+  </MainContainer>
+);
+
+const MainContainer = styled.div`
+  display: flex;
+  margin: 5em;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  ${media(
+    "mobile",
+    `
+        margin: 1em;
+        flex-direction: column-reverse;
+        `
+  )};
+  ${media(
+    "tablet",
+    `
+        margin: 1em;
+        `
+  )};
+`;
+
+const SText = styled(Text)`
+  ${({ theme }) => `
+    font-family: ${theme.font.lato};
+    color: ${theme.colors.navy};
+    margin-top: 0;
+  `};
+  ${media(
+    "mobile",
+    `
+        line-height: 1.3;
+        `
+  )};
+  ${media(
```

Comment:
this is a repeat; just have ```tablet``` and remove the other one. the media queries will be applied to anything that is smaller than the breakpoint you listed

Reply:
Didn't know that! Thanks for letting me know - I'll be sure to incorporate that